contracts/modules/capital/Pool.sol

```diff
@@ -102,13 +104,13 @@ contract Pool is IPool, MasterAwareV2, ReentrancyGuard {
 
   /* ========== ASSET RELATED VIEW FUNCTIONS ========== */
 
-  function getAssetValueInEth(address assetAddress) internal view returns (uint) {
+  function getAssetValueInEth(address assetAddress, uint amountInSwapOperator) internal view returns (uint) {
 
     uint assetBalance;
 
     if (assetAddress.code.length != 0) {
       try IERC20(assetAddress).balanceOf(address(this)) returns (uint balance) {
-        assetBalance = balance;
+        assetBalance = balance + amountInSwapOperator;
       } catch {
         // If balanceOf reverts consider it 0
       }
```

Comment:
should `amountInSwapOperator` be also added in the `if (assetBalance == 0)` condition below?

Reply:
Here we sum it with balance if it's zero and the balance is zero, assetBalance will be zero

Reply:
is it possible that all of the Pool's balance is in the SwapOperator? i.e. Pool's balance = 0, but amountInSwapOperator > 0

Reply:
It would still be gt then 0